### ISSUE TITLE
Fail running makeimage.sh

### ISSUE THREAD

**Issue by sombrowsky** [BtbN/FFmpeg-Builds#338]
I remember at my issue:
https://github.com/BtbN/FFmpeg-Builds/issues/329
that I have deleted.
I am not so firm in bash script programming.
I tried to run the scripts:
./makeimage.sh win64 gpl-shared 6.1
and
./makeimage.sh win32 gpl-shared 6.1
within a Windows 11 bash with an installed
docker.
But every times these scripts ended with an error.
I have new installed WSL and docker on
Windows 11 but no other results occures.
I used every time the scripts of the current git repository of
https://github.com/BtbN/FFmpeg-Builds.git.

Today I tried to run these scripts again, but after
I deleted the folder
scripts.d/50-vulkan
no errors occurs and then I could run the
scripts
./build.sh win64 gpl-shared 6.1
and
./build.sh win32 gpl-shared 6.1
without problems.
Could it be that some changings in
vulkan script repos are not compatible to
some configuration parameters?

**Comment by BtbN** [comment]
Works fine in all configurations for me. Without sharing any information, there's not much else I can tell you.

**Comment by BtbN** [comment]
Did you modify the scripts in any way?
I can't reproduce the issues locally, and the CI is running just fine as well.

**Comment by sombrowsky** [comment]
No I dont modify the scripts. They was the scripts from original git repository

**Comment by BtbN** [comment]
I just did a full clean build of win32 nonfree/gpl again, and it 
succeeds just fine.
Make sure you don't have any stale images around, clean out any cached 
and docker images.

**Comment by sombrowsky** [comment]
Before I run the script (only gpl-shared, not nonfree):
./makeimage.sh win64 gpl-shared 6.1

I stopped and removed all container with:
$ docker stop $(docker ps -a -q)
$ docker rm $(docker ps -a -q)

I removed all images :
$ docker rmi $(docker images -a -q)

And I removed all volumes,
but it was the same result.
I use a bash-shell (WSL) within a Windows11 Pro
with a Ubuntu 22.04.3 LTS subsystem

**Comment by BtbN** [comment]
That is win64, the log you provided was a win32 build.
Not sure what to tell you. Make sure you're on the latest version and have no stale caches, and it'll just work.
I tested all variants now, and they all work.

**Comment by sombrowsky** [comment]
Sorry, the script was:
./makeimage.sh win32 gpl-shared 6.1

I will do all for win64 with clean docker again, but I had the same result earlier.
Is there another idea to solve the problem? 
I have also tested within a VirtualBox 
ubuntu-machine, there was the same result. 
I can use the builds without 
scripts.d/50-vulkan/50-shaderc.sh
and
scripts.d/50-vulkan/60-libplacebo.sh,
but I have compiled a lot of versions before 12/2023
with good results. I used it for my free applications
in my home page https://www.somby.de.  
 
Sorry, if I keep occupied your time
